components/uniset/src/provider.rs

```diff
@@ -81,7 +81,7 @@ pub mod key {
     pub const XIDS_V1: ResourceKey = resource_key!(uniset, "XIDS", 1);
 }
 
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Clone, Hash, Eq)]
```

Comment:
It looks like you only need `Hash` and `Eq` for testing, right?  You might as well feature-gate these derives then.  When you update your branch, you should see an example of how to do that when I added Serde as optional.

Reply:
Ping

Reply:
Done.